benchmarks/experiment_configs/datasets/core50.json

```diff
@@ -0,0 +1,4 @@
+{
+    "dataset_name": "Core50",
```

Comment:
do we want to refer to it by this name or "CORe50"?

Reply:
I chose this only to avoid spelling mistakes.